android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -58,5 +58,22 @@ public static boolean isAvifImage(ByteBuffer buffer) {
    * @return true on success and false on failure. A few possible reasons for failure are: 1) Input
    *     was not valid AVIF. 2) Bitmap was not large enough to store the decoded image.
    */
-  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap);
+  public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
+    return decode(encoded, length, bitmap, 0);
```

Comment:
An alternative is to pass `android_getCpuCount()` here, for backward compatibility.

Then the new native `decode` method doesn't need to give `threads=0` a special meaning. This will avoid the potential confusion that libavif also treats `maxThreads=0` in the same way. This confusion may arise because the comment at lines 72-73 refers to the libavif documentation for `maxThreads` for more details on the `threads` parameter.

Reply:
> An alternative is to pass `android_getCpuCount()` here, for backward compatibility.
> 

`android_getCpuCount()` is a native function available through a dependent native library. it requires a bit more code to be able to call that from here.

> Then the new native `decode` method doesn't need to give `threads=0` a special meaning. This will avoid the potential confusion that libavif also treats `maxThreads=0` in the same way. This confusion may arise because the comment at lines 72-73 refers to the libavif documentation for `maxThreads` for more details on the `threads` parameter.

I have updated the comment to clarify that it is mapped to libavif's `maxThreads` only when the value is >= 0.